.circleci/real_config.yml

```diff
@@ -4612,6 +4612,19 @@ workflows:
           devcluster-config: multi-k8s.devcluster.yaml
           run-minikubes: true
 
+      - test-e2e:
+          name: test-e2e-single-k8s
+          context:
+            - dev-ci-cluster-default-user-credentials
+          requires:
+            - build-go-ee
+          parallelism: 1
+          tf2: true
+          mark: e2e_single_k8s
+          target-stage: master
+          devcluster-config: single-k8s.devcluster.yaml
+          run-minikubes: true
```

Comment:
This step starts up two minikubes even though we only need one. Since (per this PR description) the longevity of this job is (possibly?) dependent on the timeline shared-cluster fix, I'll leave it to you if you think it's worth adding a param to the `test-e2e` job that only runs a single minikube container when set!

Reply:
That's a fair point, I'd rather have this be a little more solid since the time line on the shared cluster fix is prioritized in favor of this approach for the time being.

I'll adjust the step to check if we're doing multi vs single

Reply:
I added a `multi-k8s` flag to the `e2e_multi-k8s` job